README.md

```diff
@@ -154,3 +154,20 @@ jvm_threads_states_threads{state="blocked",} 0.0
 > Health indicators [above](#health-endpoint) are added as gauge meters with name in the following pattern; health\_check\_%s, `%s` is a placeholder for service name e.g health\_check\_postgres
 
 The above configs can be updated on `opensrp.properties` file.
+
+### Redis for High Availability
+
+Redis Sentinel support is added on openSRP server web. To enable redis sentinel the following configurations were added:
+
+#### Configurations for the redis sentinel
+
+| Configuration        | Description                                                                     | Type                         | Default      |
+|----------------------|---------------------------------------------------------------------------------|------------------------------|--------------|
+| redis.sentinels      | Comma separated string of redis sentinel e.g. "localhost:26379,localhost:26380" | String                       | ""           |
+| redis.master         | Name of the set of redis instances to monitor.                                  | String                       | "mymaster"   |
```

Comment:
Could we add another column for _other options available_? or add them in the text at the bottom e.g. I see we have my master as the default or `redis.master` but what are alternative options? 

Also is that a name on AWS or IP address type entry e.t.c

Reply:
@ndegwamartin Updated.